File: IsPalindrome/IsPalindrome.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        string s = to_string(x); 
        int ub = s.length()-1;
        int lb = 0; 
        while(ub > lb) {
           if(s[lb] == s[ub]) {
               lb++;
               ub--;
           }
           else 
               return false; 
            
        }
        return true; 
    }
};

File: IsPalindrome/IsPalindrome.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        num = str(x) 
        i = 0; 
        e = len(num)-1
        if (x < 0): 
            return False
        while (i < e):
            if (num[i]!=num[e]):
                return False 
            else :
                e-=1 
                i+=1
        return True 